src/js/addRemove.js

```diff
@@ -0,0 +1,36 @@
+export const add = (list) => {
+  const inputField = document.querySelector('.input-container .task-input');
+  const spanalert = document.querySelector('.input-container .alert');
+  let count = localStorage.getItem('count') && list.length ? localStorage.getItem('count') : 1;
+  if (inputField.value === '') {
+    spanalert.textContent = 'Please, Type something to add';
+  } else {
+    spanalert.textContent = '';
+    list.push({
+      description: inputField.value,
+      completed: false,
+      index: count,
+    });
+    count += 1;
+    localStorage.setItem('count', JSON.stringify(count));
+    window.location.reload();
+  }
+  return localStorage.setItem('newList', JSON.stringify(list));
+};
+
+export const remove = (list, index) => {
```

Comment:
- [ ] It's not clear how lists can be deleted individually. When the trash icon is clicked it just vanishes and no task is deleted. Kindly ensure to improve upon this aspect

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/134749270-3c4f0da7-8e2e-4c7f-a2fb-14dc69e9f45a.png)

Reply:
Hi sir,
When I click on the icon the task will be deleted like the video in the demo here:
https://www.youtube.com/watch?v=AcUd-_Yjjqg
Can you tell me more please. because I did not understand exactly what you want me to do.
@rloterh